--- /home/rishabh/linux-linux-3.16.y/include/linux/syscalls.h	2019-09-24 01:42:19.000000000 +0530
+++ /home/rishabh/working_dirct/include/linux/syscalls.h	2019-09-28 21:54:05.521847349 +0530
@@ -867,5 +867,7 @@
 			 unsigned long idx1, unsigned long idx2);
 asmlinkage long sys_finit_module(int fd, const char __user *uargs, int flags);
 asmlinkage long sys_seccomp(unsigned int op, unsigned int flags,
+
 			    const char __user *uargs);
+asmlinkage long sys_hello(void);
 #endif
